examples/montecarlo.jl

```diff
@@ -24,7 +24,8 @@ function montecarlo(mc_eval::Function, mc_monitor::Function, reps, n_returns, po
     end
 
     if mod(pernode,pooled) != 0
-        pooled = div(pernode, commsize-1)
+        pooled = Int64(div(pernode, commsize-1))
```

Comment:
What I wanted to suggest is to calculate `pernode` differently so that its type remains integer. It's strange to calculate it as floating-point quantity.

I was just trying to figure out why the `Int64` conversion is necessary. Since it does seem necessary, and not through your changes, I don't want to request you to change things.